task-list/config/routes.rb

```diff
@@ -1,6 +1,19 @@
 Rails.application.routes.draw do
   get '/' => 'tasks#index'
-  resources :tasks
```

Comment:
I know it's a lot more typing to write out each route by hand, but soon enough we'll be able to use the `resources` helper!